backend/inspirehep/records/marshmallow/journals/ui.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from marshmallow import fields
+
+from inspirehep.records.marshmallow.journals.base import JournalsPublicListSchema
+
+class JournalsBaseSchema(JournalsPublicListSchema):
+    public_notes = fields.Raw()
+    short_title = fields.String()
+    publisher = fields.Raw()
+    journal_title = fields.Method("get_journal_title", dump_only=True)
+    urls = fields.Raw()
+
+    def get_journal_title(self, original_object):
+        """Populate the ``journal_title`` field of Journals records."""
+        journal_title = original_object.get_value("journal_title.title", default="")
```

Comment:
it is wrong, should be `get_value(original_object, "journal_title.title")` (no need for default value cause `title` is required by schema). Also, you need to import `get_value` ;)